cypress/assets/unfetch.umd.js

```diff
@@ -1,3 +1,12 @@
+/**
+ * See
+ *
+ * * https://github.com/cypress-io/cypress/issues/95
+ * * https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/stubbing-spying__window-fetch
+ * * https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/stubbing-spying__window-fetch/cypress/integration/polyfill-fetch-from-tests-spec.js
+ *
+ * for an explanation why this is currently necessary...
+ */
```

Comment:
Not possible to do something like they did in Maps instead of including a vendor lib?

Reply:
No, the maps app itself does that (see [@austins comment](https://github.com/dhis2/ui-core/pull/513#issuecomment-543113963)), which means that we'd have to do that in the ui-widgets library itself.

Reply:
We could argue about whether it's necessary at all do stub network requests.
There is another possible solution: Create scenario specific storybook scenarios and provide the expected response in the CustomDataProvider.

My preferred solution is the current way, see [my comment](https://github.com/dhis2/ui-core/pull/513#issuecomment-543117723) on the `ui-core`'s cucumber PR

Reply:
OK, let's go with your preferred way and try it out.